src/scene/scene.h

```diff
@@ -3,27 +3,36 @@
 #include <GL/glew.h>
 #include <glm/glm.hpp>
 
-#include "btBulletDynamicsCommon.h"
 #include "game_objects/game_object.h"
 #include "shaders/shader.h"
-#include "boost/signals2.hpp"
+#include "scene_info.h"
+
+// These constants should be equivalent to those in the shaders!
```

Comment:
Is it possible to get the shaders to reference these instead? If not, don't worry about it.